ui/src/components/graphs/ErrorsGraph.tsx

```diff
@@ -20,6 +21,7 @@ interface ErrorsGraphProps {
   from: number
   to: number
   uPlotCursor: uPlot.Cursor
+  updateTimeRange: (min: number, max: number, absolute: boolean) => void
```

Comment:
I guess, for less duplication and more rigidity, there can be a base type called `GraphProps` with fields like from, to, updateTiemRange and that can extended in these three components to add their fields.

Reply:
That makes sense, yes. Given this is a PR to fix a bug I won't address this now. We should definitely do this later!